src/main/java/spark/Spark.java

```diff
@@ -36,11 +31,24 @@
  *
  * @author Per Wendel
  */
-public final class Spark extends SparkBase {
+public final class Spark {
+    private static SparkInstance instance;
+
     // Hide constructor
     private Spark() {
     }
 
+    /**
+     * Initializes singleton.
+     */
+    private static class SingletonHolder {
```

Comment:
What purpose does this class serve? Why not just keep the instance at the top level?